src/pages/komodo-defi-framework/api/v20/enable_tendermint_with_assets/index.mdx

```diff
@@ -29,6 +29,10 @@ Use this method to activate Tendermint coins (COSMOS/IRIS/OSMOSIS) and IBC asset
 | tokens\_balances | array of objects | Only returned when `get_balances` is `true`. A list of standard [AddressInfo](/komodo-defi-framework/api/common_structures/#address-info) objects, one for each token. |
 | tokens\_tickers  | array            | Only returned when `get_balances` is `false`. A list of each token which was activated.                                                                                |
 
+<Note>
+ The `rpc_urls` parameter will be replaced with the `nodes` parameter in the next release. Instead of a list of url strings, the `nodes` parameter will be a list of [EvmNode objects](/komodo-defi-framework/api/common_structures/activation/#evm-node).
```

Comment:
Was the "EvmNode objects" part intentional here?

Reply:
I've duplicated the `EvmNode` object and named it `TendermintNode` in common structures for better clarity.

Reply:
`gui_auth` is no longer used. We renamed it as `komodo_proxy`

Reply:
I've updated this across the docs & postman, with a note `Prior to the v2.2.0 release, the "gui_auth" field was used to access RPC nodes run officially by the Komodo Platform team. This field was replaced with the "komodo_auth" field in the v2.2.0 release.`